job-utilities/create-eventbridge-schedule/deploy_schedule.py

```diff
@@ -4,22 +4,22 @@
 """
 import json
 import os
-import boto3
 import sys
+import boto3
+from botocore.exceptions import ClientError
 
 client = boto3.client('events')
 lambda_client = boto3.client('lambda')
 
 environment = os.getenv('CMR_ENVIRONMENT').lower()
-jobs_file = os.getenv('JOBS_FILE', '../job-details.json')
 
 if len(sys.argv) < 2:
     print("Usage is 'python deploy_schedule.py job_name [jobs_file]'")
-    exit()
+    sys.exit()
 
-if environment==None:
+if environment is None:
```

Comment:
isn't pylint fun!

Reply:
It's super helpful! And it's funny that it gives a rating